shared/src/main/scala/amf/client/remod/AMFRenderer.scala

```diff
@@ -0,0 +1,23 @@
+package amf.client.remod
+
+import amf.core.model.document.BaseUnit
+import org.yaml.model.YDocument
+
+private[remod] object AMFRenderer {
+
+  def render(bu: BaseUnit, env: AMFConfiguration): String = ???
+
+  def buildAST(bu: BaseUnit, env: AMFConfiguration): YDocument = ???
+
+  /**
+    *
+    * @param bu
+    * @param target media type which specifies a vendor, and optionally a syntax.
+    * @param env
+    * @return
+    */
+  def render(bu: BaseUnit, target: String, env: AMFConfiguration): String = ???
```

Comment:
target should be typed in Vendor for clarification

Reply:
The idea is to handled vendors using media type standard, so the users don't have to know our static objects.
We should be able to parse the media type expression/

Reply:
https://www.mulesoft.org/jira/browse/APIMF-3006

Reply:
Would rename this to media type then

Reply:
Change "target" to "mediaType"? I think could be confusing. The argument is the target to which you want to serialize, media type is the argument "type" or "format".
Adding documentation to the argument that needs to be a valid media type should be enough.
(Pending task)